include/zephyr/bluetooth/conn.h

```diff
@@ -454,6 +529,66 @@ int bt_conn_get_remote_info(struct bt_conn *conn,
 int bt_conn_le_get_tx_power_level(struct bt_conn *conn,
 				  struct bt_conn_le_tx_power *tx_power_level);
 
+#if defined(CONFIG_BT_TRANSMIT_POWER_CONTROL)
```

Comment:
Please avoid #ifdefs in header files. It prevents usage of `IS_ENABLED()` in the code and also requires explicit defines for the documentation build process.